### ISSUE TITLE
Plugin not working at all | Grafana 

### ISSUE THREAD

**Issue by PiperMp3** [cloudspout/cloudspout-button-panel#10]
Installed using readme
7.0.2 does not appear under visualisers at all.
7.0.3 and 7.0.4 does, however either of them, when selecting button panel triggers Grafana error;
An unexpected error happened
Details
Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

**Comment by dlrudwns135** [comment]
me too.
how to fix it?

**Comment by victorUTF** [comment]
Me too.

In Dependency says 6.3.X, does that mean exactly 6.3.X? Or higher than 6.3.X?
``` 
notebook@notebook:~$ grafana-server -v
Version 6.7.3 (commit: a04ef6cefc, branch: HEAD)
```

**Comment by dlrudwns135** [comment]
i use 6.6.1 version grafana. here is not working.
i test several version this plugins.
this is working good when match grafana version and plugin version.
and i check commit this plugin, this plugin upgrade for grafana version 7 after 1.0.4.
i test plugin version before 1.0.4 in grafana version 6, it is not working.

**Comment by derjust** [comment]
>i use 6.6.1 version grafana. here is not working.
the plugin use the new platform introduced with Grafana 7.0.0

> In Dependency says 6.3.X
Thank you @victorUTF for reporting. This was an oversight and is now fixed.

Looking into the error message reported by @PiperMp3

**Comment by corgan2222** [comment]
same problem here..
it did not show the rest panel

grafana 7.2.1 with 7.0.4
any ideas?

**Comment by derjust** [comment]
The issue should be fixed with the new release `7.0.5` of the plugin.
It is in review to be added to the Grafana web installer: https://github.com/grafana/grafana-plugin-repository/pull/756

Until then, a manual install via the CLI is already possible:
```BASH
$ grafana-cli --pluginUrl https://github.com/cloudspout/cloudspout-button-panel/releases/download/7.0.5/cloudspout-button-panel.zip \
  plugins install cloudspout-button-panel
```
